code/modules/clothing/spacesuits/void/zaddat_yw.dm

```diff
@@ -0,0 +1,207 @@
+/obj/item/clothing/suit/space/void/zaddat/engineer
+	name = "\improper Engineer's Guild Shroud"
+	desc = "This rugged Shroud was created by the Xozi Engineering Guild."
+	armor = list(melee = 0, bullet = 0, laser = 0,energy = 0, bomb = 0, bio = 100, rad = 100)
+	icon_state = "zaddat_engie"
+	item_state = "zaddat_engie"
+	helmet = new/obj/item/clothing/head/helmet/space/void/zaddat/engineer
+
+
+/obj/item/clothing/head/helmet/space/void/zaddat/engineer
+	name = "\improper Engineer's Guild Shroud helmet"
+	desc = "A Shroud helmet designed for good visibility in low-light environments."
+	icon_state = "zaddat_engie"
+	item_state = "zaddat_engie"
+	armor = list(melee = 10, bullet = 0, laser = 0, energy = 0, bomb = 0, bio = 100, rad = 100)
```

Comment:
```suggestion
	armor = list(melee = 0, bullet = 0, laser = 0, energy = 0, bomb = 0, bio = 100, rad = 100)
```
You forgot this or git-hub bugged out, this needs to be 0 for this to pass.

Reply:
Oh right it went just on suit, odd